### PR TITLE
refactor: [ANDROAPP-4408] remove coordinates

### PR DESCRIPTION
**Pull request by andresmr**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/src/main/res/layout-land/activity_event_initial.xml [hunk 1]
```diff
@@ -232,13 +232,6 @@
                             android:layout_height="wrap_content"
                             android:orientation="vertical" />
 
-                        <include
```
Comment (Balcan) [inline app/src/main/res/layout-land/activity_event_initial.xml:235]: Why is this deleted in the landscape layout but not in the portrait one?? I think we need it to capture coordinates in the event initial screen.

File: app/src/main/res/layout/activity_event_initial.xml [hunk 2]
```diff
@@ -224,8 +224,8 @@
                         android:orientation="vertical" />
 
                     <include
-                        android:id="@+id/geometry"
```
Comment (Balcan) [inline app/src/main/res/layout/activity_event_initial.xml:227]: Related to the previous comment. Here we are showing the coordinates field in event inital as it should happen in landscape.

File: app/src/main/java/org/dhis2/usescases/searchTrackEntity/SearchRepositoryImpl.java [hunk 3]
```diff
@@ -152,8 +151,7 @@
                 })
                 .toList().map(list ->
                         CollectionsKt.filter(list, item ->
-                                !(item instanceof PictureViewModel) &&
```
Comment (Balcan) [inline app/src/main/java/org/dhis2/usescases/searchTrackEntity/SearchRepositoryImpl.java:155]: We need to filter picture and coordinates fields so that they are not displayed in the attributes list in the TEI search card.

File: app/src/main/java/org/dhis2/data/forms/dataentry/FormView.kt [hunk 4]
```diff
@@ -323,10 +320,20 @@ class FormView constructor(
         imm?.hideSoftInputFromWindow(binding.recyclerView.windowToken, 0)
     }
 
-    private fun doesItemNeedsKeyboard(item: FieldUiModel) = when (item) {
```
Comment (Balcan) [inline app/src/main/java/org/dhis2/data/forms/dataentry/FormView.kt:326]: ValueType enum has some utility methods like `isText` which you may use here instead of having such a big list

**Review by mmmateos — APPROVED** [review]

File: app/src/main/java/org/dhis2/data/forms/dataentry/FormView.kt (hunk 4 above)
Comment (andresmr) [reply app/src/main/java/org/dhis2/data/forms/dataentry/FormView.kt:326]: Very good tip!, thanks.

File: app/src/main/java/org/dhis2/usescases/searchTrackEntity/SearchRepositoryImpl.java (hunk 3 above)
Comment (andresmr) [reply app/src/main/java/org/dhis2/usescases/searchTrackEntity/SearchRepositoryImpl.java:155]: check if this fits

**Review by Balcan — APPROVED** [review]
